CMP1903_A1_2324/Game.cs

```diff
@@ -3,19 +3,59 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Threading;
 
 namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        // Instantiate 3 objects of the die
+        Die die1 = new Die();
```

Comment:
Shows use of creating objects, using the Die class.

Reply:
Thank you!